tests/spec/core/data-tests-spec.js

```diff
@@ -63,5 +71,78 @@ describe("Core — data-tests attribute", () => {
         expect(pathname.endsWith(`test${i}.html`)).toBe(true);
       });
     });
+    it(`adds emojis for secure tests`, () => {
+      const li = doc.querySelector("#metadata-tests ul>li:nth-child(1)");
+
+      const fileName = li.children[0].innerText.trim();
+      expect(fileName).toEqual("this-is-secure");
+
+      const link = li.children[0].href;
+      expect(link).toEqual("https://wpt.fyi/respec/this-is-secure.https.html");
+
+      const emoji = li.children[1].innerText;
+      expect(emoji).toEqual("🔒");
+
+      const spanAriaLabel = li.children[1].children[0].getAttribute(
+        "aria-label"
+      );
+      expect(spanAriaLabel).toEqual("requires a secure connection");
+
+      const spanTitle = li.children[1].children[0].getAttribute("title");
+      expect(spanTitle).toEqual("Test requires HTTPS");
+    });
+    it(`adds emojis for manual tests`, () => {
+      const li = doc.querySelector("#metadata-tests ul>li:nth-child(2)");
+
+      const fileName = li.children[0].innerText.trim();
+      expect(fileName).toEqual("this-is");
+
+      const link = li.children[0].href;
+      expect(link).toEqual("https://wpt.fyi/respec/this-is-manual.html");
+
+      const emoji = li.children[1].innerText;
+      expect(emoji).toEqual("💪");
+
+      const spanAriaLabel = li.children[1].children[0].getAttribute(
+        "aria-label"
+      );
+      expect(spanAriaLabel).toEqual("the test must be run manually");
+
+      const spanTitle = li.children[1].children[0].getAttribute("title");
+      expect(spanTitle).toEqual("Manual test");
+    });
+    it(`adds emojis for secure manual tests`, () => {
+      const li = doc.querySelector("#metadata-tests ul>li:nth-child(3)");
+
+      const fileName = li.children[0].innerText.trim();
+      expect(fileName).toEqual("this-is-secure-and");
+
+      const link = li.children[0].href;
+      expect(link).toEqual(
+        "https://wpt.fyi/respec/this-is-secure-and-manual.https.html"
+      );
+
+      const emojiSecure = li.children[1].children[0].innerText;
+      expect(emojiSecure).toEqual("🔒");
+
+      const spanAriaLabel1 = li.children[1].children[0].getAttribute(
+        "aria-label"
+      );
+      expect(spanAriaLabel1).toEqual("requires a secure connection");
+
+      const spanTitle1 = li.children[1].children[0].getAttribute("title");
+      expect(spanTitle1).toEqual("Test requires HTTPS");
+
+      const spanEmoji2 = li.children[1].children[1].innerText;
+      expect(spanEmoji2).toEqual("💪");
+
+      const spanAriaLabel2 = li.children[1].children[1].getAttribute(
+        "aria-label"
+      );
+      expect(spanAriaLabel2).toEqual("the test must be run manually");
+
+      const spanTitle2 = li.children[1].children[1].getAttribute("title");
```

Comment:
Seems odd that they are nested? I think this might be a bug. The tree structure should be:

```JS
<span>file-name</span> <span>emoji</span> <span>emoji</span>
```

Or am I reading this wrong?

Reply:
Its something like 

`<li><a>fileName</a><span>emoji</span></li>`4

So, yeah roughly the same